config/bidderinfo.go

```diff
@@ -591,6 +595,42 @@ func validatePlatformInfo(info *PlatformInfo) error {
 	return nil
 }
 
+func validateGeoscope(geoscope []string, bidderName string) error {
+	if len(geoscope) == 0 {
+		return nil
+	}
+
+	// ISO 3166-1 alpha-3 country codes are uppercase 3-letter codes
+	for i, code := range geoscope {
+		code = strings.ToUpper(strings.TrimSpace(code))
+
+		if code == "GLOBAL" || code == "EEA" {
+			continue
+		}
+
+		// Handle exclusion pattern with "!" prefix
+		exclusion := ""
+		if strings.HasPrefix(code, "!") {
+			exclusion = "!"
+			code = strings.TrimPrefix(code, "!")
```

Comment:
Nitpick: To remove the first character, seems more natural to use `code = code[1:]`.